.github/workflows/continuous-integration.yml

```diff
@@ -312,11 +366,34 @@ jobs:
             .cache/yarn
             node_modules
 
+      - name: List changed files
+        id: changed-files
+        run: echo ::set-output name=all_changed_files::$(git diff-tree --no-commit-id --name-only -r ${{ github.sha }})
+
+      - name: Get app folders for changed files
+        id: get-changed-apps
+        run: echo ::set-output name=app_folders::$(node script/github-actions/get-changed-apps.js --output-type folder | tr ',' ' ')
+        env:
+          CHANGED_FILE_PATHS: ${{ steps.changed-files.outputs.all_changed_files }}
+
       - name: Annotate ESLint results
-        run: yarn run eslint --ext .js --ext .jsx --format ./script/github-actions/eslint-annotation-format.js .
+        run: |
+          if [[ -z "${{ steps.get-changed-apps.outputs.app_folders }}" ]]; then
+            yarn run eslint --ext .js --ext .jsx --format ./script/github-actions/eslint-annotation-format.js .
+          else
+            yarn run eslint --ext .js --ext .jsx --format ./script/github-actions/eslint-annotation-format.js \
+            ${{ steps.get-changed-apps.outputs.app_folders }}
+          fi
 
       - name: Run Stylelint
-        run: yarn run stylelint verbose --output-file stylelint-report.json --formatter json src/**/*.scss
+        if: ${{ always() }}
```

Comment:
Currently, we're always running the `Annotate Stylelint results` step without always running this step, resulting in there being nothing to annotate if the ESLint step fails, so I added this here to ensure Stylelint always runs.

It would be nice if we could consolidate the two into one step to make Stylelint failures easier to read, similar to ESLint, but that's beyond the scope of this PR